packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/aspnet/configmid.md

```diff
@@ -38,6 +38,26 @@ public class Startup
 }
 ```
 
+The Okta middleware uses the `default` Authorization Server by default. If you are using a [custom Authorization Server](https://developer.okta.com/docs/concepts/auth-servers/), make sure to configure it by setting up the `AuthorizationServerId`. For example, if you are using the Org Authorization Server your middleware configuration should look like this: 
```

Comment:
- Don't need "The" at the beginning - just start with Okta middleware uses.....

- capitalize the c in Custom ->[Custom Authorization Server]

- you can shorten the link to: (/docs/concepts/auth-servers/)

comma before your in the last sentence:
......Org Authorization Server, your middleware configuration....

Question: The `default` Authorization Server is a custom authorization server. Does Okta middleware use the `default` Custom Authorization Server or the Org Authorization Server? In the last sentence of the paragraph, the Org Authorization Server is referenced as an example. But, the second to the last sentence is talking about configuring the `AuthorizationServerId` for a Custom Authorization Server.

I just want to be sure that I understand which authorization server is being used where - because it's actually super confusing for users to understand.

Org Authorization Server URL is just: https://${yourOktaOrg}
Default Custom Authorization Server URL is: https://${yourOktaDomian}/api/v1/authorizationServers/default
And any Custom Authorization Server that the user creates is: https://${yourOktaDomain}/api/v1/authorizationServers/${authServerId}

Reply:
Good point! Okta middleware uses the `default` Custom Authorization Server. I'll reword that part to clarify this better.